client/plots/regression.results.js

```diff
@@ -470,7 +470,7 @@ function setRenderers(self) {
 		{
 			const tr = table.append('tr').style('opacity', 0.4)
 			result.coefficients.header.forEach((v, i) => {
-				tr.append('td').text(v).style('padding', '8px')
+				tr.append('td').text(v).style('padding', '8px').attr('data-testid', 'sja_results')
```

Comment:
move testid to table on line 465. name testid as `sjpp_regression_resultCoefficientTable`
`sja_results` is too general to be proper testid